install/streams.csv

```diff
@@ -2512,8 +2512,9 @@ WSTS,21,50,false,false,false,2016-12-06T03:30:00Z,2020-06-16T00:00:00Z
 WSTS,21,200,false,false,true,2016-12-06T03:30:00Z,9999-01-01T00:00:00Z
 WTA,10,50,false,false,true,1990-02-19T21:40:00Z,1990-05-17T20:29:00Z
 WTAC,10,50,false,false,true,2001-01-12T08:37:00Z,2001-06-26T19:07:00Z
-WTAZ,10,100,false,false,false,2006-12-13T02:20:01Z,9999-01-01T00:00:00Z
+WTAZ,10,100,false,false,false,2006-12-13T02:20:01Z,2022-05-17T01:32:00Z
 WTAZ,11,100,false,false,false,2007-05-07T01:00:00Z,2007-05-14T02:30:00Z
+WTAZ,12,100,false,false,false,2022-05-24T22:54:00Z,9999-01-01T00:00:00Z
```

Comment:
As is is now "technically" a broadband site rather than short period there will be 1Hz LHZ | N | E streams produced. Worth adding?

 ```
 NZ WTAZ  12 HHE D 2022-06-07 14:52:34  -  2022-06-07 22:34:40
NZ WTAZ  12 HHN D 2022-06-07 14:52:32  -  2022-06-07 22:34:39
NZ WTAZ  12 HHZ D 2022-06-07 14:52:32  -  2022-06-07 22:34:38
NZ WTAZ  12 LHE D 2022-06-07 14:49:06  -  2022-06-07 22:31:35
NZ WTAZ  12 LHN D 2022-06-07 14:48:16  -  2022-06-07 22:30:37
NZ WTAZ  12 LHZ D 2022-06-07 14:51:30  -  2022-06-07 22:33:33
```

Reply:
yo @CBurton90 indeed! since it is provide as a default configuration. - i forgot - i ll repent and amend! 
correcting soon